libs/dh/admin/feature-user-roles/src/lib/edit/dh-edit-user-role-modal.component.ts

```diff
@@ -149,7 +158,7 @@ export class DhEditUserRoleModalComponent implements OnInit, AfterViewInit {
         permissionIds,
       });
 
-      this.marketRolePermissionsStore.getPermissions(userRole.eicFunction);
+      this.permissionsQuery.refetch({ eicFunction: userRole.eicFunction });
```

Comment:
Suggestion: This could be moved into a constructor and an effect, to get rid of the ngOnInit